crates/node_binding/binding.d.ts

```diff
@@ -13,33 +13,33 @@ export type RawLazyCompilationTest = RegExp | ((module: Module) => boolean);
 export type AssetInfo = KnownAssetInfo & Record<string, any>;
 
 export interface Module {
-	readonly type: string;
-	get context(): string | undefined;
-	get layer(): string | undefined;
-	get factoryMeta(): JsFactoryMeta | undefined
-	get useSourceMap(): boolean;
-	get useSimpleSourceMap(): boolean;
-	buildInfo: Record<string, any>;
-	buildMeta: Record<string, any>;
+  readonly type: string;
```

Comment:
seems your format is not same

Reply:
i will update this later, i finish the api align and will add some test case for this~